### PR TITLE
[PHP 8.0] Add promoted properties in Page, Notification, Lead and Plugin bundles - step 4

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: app/bundles/LeadBundle/EventListener/LeadSubscriber.php [hunk 1]
```diff
@@ -29,52 +29,30 @@ class LeadSubscriber implements EventSubscriberInterface
...
-        DncReasonHelper $dncReasonHelper,
-        EntityManager $entityManager,
-        TranslatorInterface $translator,
+        private IpLookupHelper $ipLookupHelper,
+        private AuditLogModel $auditLogModel,
+        private LeadChangeEventDispatcher $leadEventDispatcher,
+        private DncReasonHelper $dncReasonHelper,
+        private EntityManager $entityManager,
+        private TranslatorInterface $translator,
         RouterInterface $router,
```
Comment (escopecz) [inline app/bundles/LeadBundle/EventListener/LeadSubscriber.php:48]: @TomasVotruba do you know why Rector omitted this?

File: app/bundles/LeadBundle/EventListener/TypeOperatorSubscriber.php [hunk 2]
```diff
@@ -31,39 +31,18 @@ final class TypeOperatorSubscriber implements EventSubscriberInterface
...
         TranslatorInterface $translator
     ) {
-        $this->leadModel     = $leadModel;
-        $this->listModel     = $listModel;
-        $this->campaignModel = $campaignModel;
-        $this->emailModel    = $emailModel;
-        $this->stageModel    = $stageModel;
-        $this->categoryModel = $categoryModel;
-        $this->assetModel    = $assetModel;
         $this->translator    = $translator;
```
Comment (escopecz) [inline app/bundles/LeadBundle/EventListener/TypeOperatorSubscriber.php:46]: The translator could also be defined in the constructor, right? I'm not certain what I'm missing

File: app/bundles/LeadBundle/EventListener/LeadSubscriber.php (hunk 1 above)
Comment (TomasVotruba) [reply app/bundles/LeadBundle/EventListener/LeadSubscriber.php:48]: Not sure, I'd suggest to skip it now. The last PR will add this rule in CI and will enable for all the files. Will check then

File: app/bundles/LeadBundle/EventListener/TypeOperatorSubscriber.php (hunk 2 above)
Comment (TomasVotruba) [reply app/bundles/LeadBundle/EventListener/TypeOperatorSubscriber.php:46]: Not sure. I'd suggest to skip it now, I'll handle it later

**Comment by escopecz** [comment]
@TomasVotruba it seems that we are missing a step? Rector is reporting many issues

**Comment by TomasVotruba** [comment]
I'll pull and rebase :+1:

**Comment by TomasVotruba** [comment]
> @TomasVotruba it seems that we are missing a step? Rector is reporting many issues

I accidentaly left the rule in `rector.php`. It should have been removed, as more PRs to come with this rule.
Fixed now :+1:

**Comment by TomasVotruba** [comment]
Good to go :heavy_check_mark:  :partying_face:

**Review by escopecz — APPROVED** [review]
👍

**Pull request merged**